deploy/kubernetes/snapshot/validation-webhook/README.md

```diff
@@ -0,0 +1,62 @@
+3. Fetch the certificate authority of the cluster where you are deploying the webhook. You will either find it or tha path leading to it in your KUBECONFIG file, under the name `certificate-authority` under your cluster. Fetch the content of the file (it begins and ends with `-----BEGIN CERTIFICATE-----` and `-----END CERTIFICATE-----` respectively). Encode the file `base64 -i <file>.pem`.
+4. Add the base64 encoded value in `deploy/kubernetes/webhook-example/admission-configuration.yaml` file in front of `caBundle:` paramater.
+5. Change the namespace to where you prefer to deploy the webhook in `admission-configuration.yaml, rbac-snapshot-webhook.yaml, webhook.yaml` files under `deploy/kubernetes/webhook-example`/` directory
+6. Run `kubectl create -f webhook-example`
+7. Once done, you should be able to see 3 pods, 1 service and 1 secret in the provided namespace as shown below.
+```
+% kubectl get pods -n kube-system | grep snapshot-validation
+snapshot-validation-deployment-6f88cc8b87-fdl4p       1/1     Running     0                6d16h
+snapshot-validation-deployment-6f88cc8b87-jlc5g       1/1     Running     0                6d16h
+snapshot-validation-deployment-6f88cc8b87-mmbd2       1/1     Running     0                6d16h
+
+% kubectl get svc -n kube-system | grep snapshot-validation
+snapshot-validation-service          ClusterIP      <CLUSTER-IP>   <none>                               443/TCP                                                       12d
+
+% kubectl get secrets -n kube-system | grep snapshot-validation
+snapshot-validation-secret                         Opaque                                2      6d16h
+```
+8. Check pod logs. 
+- If you see the following, web hook is up and running
+    ```
+    kubectl logs -f snapshot-validation-deployment-6f88cc8b87-fdl4p -n kube-system
+    I0921 18:15:14.545269    1 certwatcher.go:129] Updated current TLS certificate
+    Starting webhook server
+    I0921 18:15:14.645808    1 webhook.go:196] Starting certificate watcher
+    ```
+- If the following logs are observed, 
+    ```
+    2022/09/22 18:38:49 http: TLS handshake error from <IP>:<PORT>: remote error: tls: bad certificate
+    2022/09/22 18:38:50 http: TLS handshake error from <IP>:<PORT>: remote error: tls: bad certificate
+    2022/09/22 18:38:51 http: TLS handshake error from <IP>:<PORT>: remote error: tls: bad certificate
+    2022/09/22 18:38:52 http: TLS handshake error from <IP>:<PORT>: remote error: tls: bad certificate
+    2022/09/22 18:38:53 http: TLS handshake error from <IP>:<PORT>: remote error: tls: bad certificate
+    ```
+    The k8s secret used by webhook is not populated with the cert and key value. 
+    OR
+    The certificate provided in the `caBundle:` may not be right. If neither is the case, reach out to the community - raise an issue [here](https://github.com/kubernetes-csi/external-snapshotter/issues).
+
+### Deploying webhook with cert manager
```

Comment:
Can we add a note if this is not finalised

Reply:
I added the doc